test/titleFormatter.test.ts

```diff
@@ -0,0 +1,315 @@
+import { formatTitle, TitleFormatting } from "../src";
+import { toCapitalizeCase, toTitleCase, toSentenceCase } from "../src/formatters";
+import { capitalizeFirstLetter, cleanPunctuation, isAcronym, isAcronymStrict, isInTitleCase, isMostlyAllCaps } from "../src/helpers";
+
+// TODO: split tests across multiple files
```

Comment:
This is more of a tip. Since we have a completely new repo, it might be worth organizing these tests in the future too 😅